### PR TITLE
Aquilon installation: various refinements

### PR DESCRIPTION
**Pull request by jouvin**
* Miscellaneous changes in the template examples, based on LAL experience
* Document AII installack configuration for Aquilon, in particular when sharing an AII server between SCDB and Aquilon

### CODE REVIEW

**Comment by jrha** [comment]
Tests are failing due to what looks like issues with GitHub.

**Comment by jouvin** [comment]
Test failure is due to spellchecker issue https://github.com/stfc/markdown-spellchecker/issues/11. @jrha do you prefer that I add the necessary entries in the dictionary to make the spell checker happy or do we live with these errors until the issue is fixed (probably better than polluting the dictionary if there is any hope to fix the issue in the coming months...

**Comment by jrha** [comment]
I was referring to the link errors from htmlproofer:
>  *  External link https://github.com/quattor/configuration-modules-core/pull/129 failed: 500 No error

There is one *real* spell checker error, feel free to ignore the false ones, it's probably better not to pollute the dictionary, the fix should be trivial.

**Comment by jouvin** [comment]
@jrha sorry I didn't see your original comment, mine was not an answer to yours, just a general remark! I think I fixed the real error but I let you confirm that the ones remaining are ok.

File: _aquilon/configuration.md [hunk 1]
```diff
@@ -686,7 +686,11 @@ cd /var/quattor/templates/$USER/tutorial
 cat > web_servers/archetype/base.pan <<EOF
 unique template archetype/base;
 
-# This variable will allow to control the YUM snapshot used, if you are using YUM
+# Define the default suffix for YUM-based variant of RPMS configuration templates
+# Defined too late by the standard templates in the context of Aquilon
+variable RPMS_CONFIG_SUFFIX ?= '-yd';
```
Comment (ned21) [inline _aquilon/configuration.md]: I've never seen this variable before so shouldn't really comment on it but -yd seems an odd suffix to me?

**Comment by ned21** [comment]
Most of the content here is related to parts of Quattor we don't use but nothing looks incorrect from an Aq perspective.

File: _aquilon/configuration.md (hunk 1 above)
Comment (jouvin) [reply _aquilon/configuration.md]: @ned21 this variable is used by the template library as a template name suffix in various places where we wanted to support both the SPMA package list and a YUM-based package list. For backward compatibility, without the suffix you were getting the SPMA list and you needed to have this variable defined to `-yd` to get the YUM-based list. I don't even remember why `-yd` (`y` was probably for `YUM`). I think we may clean up this in the future as I don't think there are many non YUM system left nowadays but it is important to have this variable defined if you use the template library. Else several pieces don't work as expected.

Comment (jouvin) [reply _aquilon/configuration.md]: The comment about the standard definition being done to late is due to the fact that this variable is defined in `quattor/client/config` which is included in `final.pan` with Aquilon (as it has a dependency on the OS version variable which is not yet available in `base.pan` (typically defined in the OS template).

**Comment by jouvin** [comment]
@jrha @ned21 anything I should do to get this PR merged?

**Comment by ned21** [comment]
@jouvin I am not familiar with much of the material here so would prefer someone else to do the final review.

**Pull request merged**